### PR TITLE
[extraction] Cleanup open modules

### PR DESCRIPTION
**Pull request by Alizter**
We clean up some open modules in extraction.

### CODE REVIEW

**Review by SkySkimmer — CHANGES_REQUESTED** [review]
This is more uglify than cleanup IMO
Especially Util and Names should stay open

File: plugins/extraction/mlutil.mli [hunk 1]
```diff
@@ -8,25 +8,21 @@
...
 (*s Utility functions over ML types with meta. *)
 
 val reset_meta_count : unit -> unit
-val new_meta : 'a -> ml_type
+val new_meta : 'a -> Miniml.ml_type
 
-val type_subst_list : ml_type list -> ml_type -> ml_type
-val type_subst_vect : ml_type array -> ml_type -> ml_type
+val type_subst_list : Miniml.ml_type list -> Miniml.ml_type -> Miniml.ml_type
+val type_subst_vect : Miniml.ml_type array -> Miniml.ml_type -> Miniml.ml_type
```
[the remark this answers is not in the document]
Comment (Alizter) [reply plugins/extraction/mlutil.mli:17]: I tried closing as much as possible in the mlis so the signatures are easier to read.

[the remark this answers is not in the document]
Comment (ppedrot) [reply plugins/extraction/mlutil.mli:17]: IMO adding redundant qualification is not making it easier to read but I know there are various schools on the topic.

**Comment by Alizter** [comment]
@SkySkimmer I can keep some modules open, but do you think the cleanup is worth it? Otherwise I won't bother too much.

**Comment by SkySkimmer** [comment]
It's probably not worth your time IMO

**Comment by Alizter** [comment]
Closing this for now

**Pull request closed without merging**